src/shared/db.py

```diff
@@ -2,10 +2,11 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-from typing import Optional, Any, List, cast, Type, TYPE_CHECKING, Union
+from typing import Optional, Any, List, Dict, cast, Type, TYPE_CHECKING, Union
 
```

Comment:
these two blocks of external library imports should be joined into one single block

Reply:
only one line and cleaned up.